contracts/governance/Staking/Staking.sol

```diff
@@ -774,4 +939,13 @@ contract Staking is
                 block.number;
         require(notSameBlock, "S20"); //S20 : "cannot be mined in the same block as last stake"
     }
+
+    /**
+     * @notice Max iteration for vesting withdrawal to prevent out of gas issue.
+     *
+     * @return max iteration value.
+     */
+    function getMaxVestingWithdrawIterations() public pure returns (uint256) {
+        return 50;
```

Comment:
pls make it a settable storage var
the new modular system will consume a bit more gas
so this value might become irrelevant at some point or under specific conditions (e.g. staking withdraw logic changes) and it looks like an overkill to update this value via a SIP